src/Host/Protocol/Impl/RUserProfileCreator.cs

```diff
@@ -83,7 +83,7 @@ public class RUserProfileCreator {
 
             string json = Encoding.Unicode.GetString(requestRaw, 0, bytesRead);
 
-            var requestData = JsonConvert.DeserializeObject<RUserProfileCreateRequest>(json);
+            var requestData = JsonConvert.DeserializeObject<RUserProfileCreateRequest>(json, new SecureStringJsonConverter());
```

Comment:
You can also stick `JsonConverterAttribute` onto the property in the class. Then you won't have to repeat the converter on every call to `Serialize` or `Deserialize`.